docs/Developer's ReadMe.md

```diff
@@ -0,0 +1,209 @@
+# Install, Build, and Test the Extension
+
+Developers can install the Visual Studio Code Extension for Zowe, which lets users interact with z/OS data sets on a remote mainframe instance, from a VSIX file and run system tests on the extension.
+
+## Contents
+
+- [Install to VSC from source](#install-to-vsc-from-source)
+- [Run System Tests](#run-system-tests)
+- [Run Zowe Explorer Regression Tests for Theia](#run-zowe-explorer-regression-tests-for-theia)
+- [Localization](#localization)
+
+## Install to VSC from source
+
+You can build the extension (VSIX file) from this source repository and install it to VSC.
+
+**Important!** Follow the [instructions for creating testProfileData.ts](#run-system-tests) before performing these steps.
+
+### Build the extension
+
+From your local copy of this repository, issue the following commands to create the VSIX package file from source:
+
+1. `yarn install`
+2. `yarn run package`
+   This creates a `.vsix` file in your local copy of the project.
+
+### Install the extension to VSC
+
+After you create a VSIX file, install the extension to VSC:
+
+1. Navigate to the Extensions menu in Visual Studio Code and click the **...** menu on the top-left.
+2. Select Install from VSIX and select the `.vsix` file that was created by your `yarn run package` command.
```

Comment:
same here: `yarn workspace vscode-extension-for-zowe package` or could this just refer to previous section?